test/topology/test_mutation_schema_change.py

```diff
@@ -20,6 +20,7 @@
 
 
 @pytest.mark.asyncio
+@pytest.mark.enable_tablets(False)
```

Comment:
Do we understand why they are not supported with Tablets? When will we enable them again? How can we track this?

Reply:
Added a comment.